lib/core/imports/index.js

```diff
@@ -47,5 +47,6 @@ axe.imports = {
 	CssSelectorParser: require('css-selector-parser').CssSelectorParser,
 	doT: require('@deque/dot'),
 	emojiRegexText: require('emoji-regex'),
-	memoize: require('memoizee')
+	memoize: require('memoizee'),
+	ariaQuery: require('aria-query')
```

Comment:
I don't think we should pull aria-query into axe-core at this point. There's a much simpler fix for this. Please don't overthink it.